tikv-configuration-file.md

```diff
@@ -1511,6 +1523,18 @@ Configuration items related to `rocksdb.defaultcf`, `rocksdb.writecf`, and `rock
 + Default value: `"128MB"`
 + Unit: KB|MB|GB
 
+### `format-version` <span class="version-mark">New in v6.3.0</span>
```

Comment:
```suggestion
### `format-version` <span class="version-mark">New in v6.2.0</span>
```
Keep it consistent with [releases/release-6.2.0.md](https://github.com/pingcap/docs/pull/13582/files#diff-d29bf2d43d9f4861e58933e6deea892bc12416c947be01e43ed742443c64f396R286)